tests/flytekit/unit/core/test_array_node_map_task.py

```diff
@@ -255,8 +260,8 @@ def many_inputs(a: int, b: str, c: float) -> str:
     m = map_task(p3)
     assert m.python_interface.inputs == {"a": int, "b": str, "c": float}
     assert (
-        m.name
-        == "tests.flytekit.unit.core.test_array_node_map_task.map_many_inputs_5d2500dc176052a030efda3b8c283f96-arraynode"
+         m.name
+         == "tests.flytekit.unit.core.test_array_node_map_task.map_many_inputs_5d2500dc176052a030efda3b8c283f96-arraynode"
```

Comment:
why are we changing this?

Reply:
it seems my editor is changing it. May be we have a formatting guide which i can import to not do this automatically in the future for my Pycharm IDE